### ISSUE TITLE
Using numeric indices in add_css_column for rownames does not apply style

### ISSUE THREAD

**Issue by clemens-zauchner** [LyzandeR/tableHTML#46]
By looking into a question on [stackoverflow](https://stackoverflow.com/questions/64884928/color-trouble-with-rmarkdown-generated-html-tables), I found a bug in `add_css_colum`: when you use the numeric index `0` to apply a style to the rownames, no inline css is applied to the respective `tr` tag, but with a character vector it works:
```r
library(tableHTML)
test <- test <- data.frame(A=123, B=456, C=789)

tableHTML(test) %>%
    add_css_column(css = list(c('border'),```
                              c("1px solid #4daf4a")),
                   columns = c("rownames", names(test))) 
```
Here, the style is applied as expected:
```html
<tbody>
<tr>
  <td id="tableHTML_rownames" style="border:1px solid #4daf4a;">1</td>
  <td id="tableHTML_column_1" style="border:1px solid #4daf4a;">123</td>
  <td id="tableHTML_column_2" style="border:1px solid #4daf4a;">456</td>
  <td id="tableHTML_column_3" style="border:1px solid #4daf4a;">789</td>
</tr>
</tbody>
```
but with the numeric index:

```r
tableHTML(test) %>%
    add_css_column(css = list(c('border'),
                              c("1px solid #4daf4a")),
                   columns = 0:4)
```

the inline css is missing:

```html
<tbody>
<tr>
  <td id="tableHTML_rownames">1</td>
  <td id="tableHTML_column_1" style="border:1px solid #4daf4a;">123</td>
  <td id="tableHTML_column_2" style="border:1px solid #4daf4a;">456</td>
  <td id="tableHTML_column_3" style="border:1px solid #4daf4a;">789</td>
</tr>
</tbody>
```

**Comment by LyzandeR** [comment]
Good Find! I'm on it!

**Comment by clemens-zauchner** [comment]
Fixed with version 2.1.0
